NfsDiagnostics/trace-nfsbpf

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
```

Comment:
IIRC, Python3 is not available by default on at least some of the distros that we support (RHEL 7?). It might be a better idea to invoke Python as `usr/bin/env python`. IIRC the scripts work the same across Python 2 & 3.